instruqt/delivering-as-an-appliance/01-specifying-the-cluster/assignment.md

```diff
@@ -1,26 +1,223 @@
+Kubernetes to embed. There are [more options for the
+configuration](https://docs.replicated.com/reference/embedded-config), but
+that's all you need to get started.
+
+The Replicated Release
+======================
+
+The Replicated Platform distributes your software to your customers. To do
+this, it needs to know about your application, it's customers, and the files
+you're shipping to them. We talk about those files as a release.
+
+Every release is built around a Helm chart, and that's all it needs. If you've
+completed the [Distributing Your Application with
+Replicated](https://play.instruqt.com/replicated/tracks/distributing-with-replicated)
+lab, you built a release around a Helm chart and installed it using Helm tools.
+To use other installation methods supported by the Reoplicated Platform, like
+the Kubernetes appliance model we're working on, you supplement the Helm chart
+with configuration files like the one we saw above.
+
+We need a minimum of two configuration files: the Embedded Cluster
+configuration and a file that describes which Helm chart to install. We'll put
+those in a directroy with the Helm chart and create a release from those files.
+
+Our Initial Appliance
+=====================
+
+Let's create a simple Kubernetes appliance for Slackernews and release it with
+the Platform. We're going to add our Helm chart and two configuration files to
+the `release` directory. These are the bare minimum set of files we need to
+create the appliance.
+
+### Bumping the Chart Version
+
+Strictly speaking, we don't have to bump the version of our Helm chart since we
+haven't changed it. It's a good practice to increment it, though, since we want
+to have consistent versions between the Helm release and the Replicated
+release. That way, regardless of the install mechanism your uses choose the
+versions will be consistent.
+
+This lab uses the application [Slackernews](https://slackernews.io) and the
+Helm sources for it are in the `slackernews` directory. You can open
+`slackernews/Chart.yaml` in the "Release Editor" tab and update the `version`.
+
+![Updating the version of the Helm chart to 0.6.0](../assets/updating-the-chart-version.png)
+
+### Adding a Helm Chart to a Release
+
+To prepare the release, we first need to make sure our Helm chart is configured
+as part of it. Let's package the chart and include it in the
+`release` directory, where we'll also add the additional files we need.
+
+```
+helm package -u slackernews -d release
+```
+
+We're then going to add a file that lets the Admin Console know about the Helm
+chart. It uses this file to identify which chart to install and pass the
+appropriate values. We'll look at the passing values in a later section of the
+lab when we set up the configuration screen for our application.
+
+Go to the "Release Editor" tab to add a file to your release.
+
+![Creating a manifest file describing your Helm chart](../assets/creating-the-helmchart-object.png)
```

Comment:
Maybe the user is supposed to open this image to see how to create the file and what to name it, but would help to at least write the name out, maybe that it belongs in the release directory.

Reply:
we mention those things later for the EC config, which was nice

Reply:
I caught his myself this morning as well.